homeassistant/components/knx/light.py

```diff
@@ -86,7 +86,9 @@ def color_temp(self):
         """Return the color temperature in mireds."""
         if self._device.supports_color_temperature:
             kelvin = self._device.current_color_temperature
-            if kelvin is not None:
+            # Some KNX-Dali gateways report 0 kelvin if the value has not been set
+            # After setting the value for the first time everything looks as expected
```

Comment:
```suggestion
            # Avoid division by zero if actuator reported 0 Kelvin (eg. uninitialized DALI-Gateway)
```
We imply that everything runs as expected. No need to comment it 😉
I think we can assume this.